roles/rabbitmq/templates/etc/serverspec/rabbitmq_spec.rb

```diff
@@ -0,0 +1,73 @@
+require 'spec_helper'
+require 'socket'
+
+describe user('rabbitmq') do
+    it { should exist }
+    it { should belong_to_group 'rabbitmq' }
+end  
+
+describe file('/etc/nova/nova.conf') do
+  it { should_not contain /(rabbit_password=\w{0,7})$/ }
```

Comment:
What is this test here? Why are you checking a nova config file in a rabbit server spec?